configapi/api_slice_mgmt_test.go

```diff
@@ -90,6 +91,23 @@ const DEVICE_GROUP_CONFIG = `{
   "site-info": "string"
 }`
 
+func networkSliceInvalidGnb(gnbName string, gnbTac int32) string {
```

Comment:
Could we change this name to something else. I think this function is creating Network Slice with whatever gNB information is passed. Why should it be called InvalidGnb?